requirements.txt

```diff
@@ -1 +1,4 @@
 pytest >= 6.0
```

Comment:
Last week, Paul showed us the "~=" syntax, which enforces a major version (e.g. pytest ~= 6.0 means 6.0<= pytest < 7.0). He described this as a preferred approach since major releases (changing x in version x.y.z) represent API changes in the code. With that said, I think we should try to convert these from ">=" to "~="